instrumentation/opentelemetry_ecto/test/opentelemetry_ecto_test.exs

```diff
@@ -111,18 +111,47 @@ defmodule OpentelemetryEctoTest do
     attach_handler()
 
     try do
-      Repo.all(from u in "users", select: u.non_existant_field)
+      Repo.all((from u in "users", select: u.non_existant_field), [stacktrace: true])
```

Comment:
We can set this in `instrumentation/opentelemetry_ecto/config/test.exs` to be a bit more clean.